@xen-orchestra/lite/src/stories/web-core/ui/table-pagination/ui-table-pagination.story.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('total-items').required().preset(100).num().widget(),
+      prop('disabled').bool().widget(),
+      event('change'),
```

Comment:
```suggestion
      event('change').args({ payload: 'PaginationPayload' }),
```

You can also add `ui-table-pagination.story.md` beside `ui-table-pagination.story.vue` just containing the typing as reference (in a TypeScript fenced code block)

Reply:
Please use a `typescript` code block as asked, not `vue-script`

Reply:
Done